### ISSUE TITLE
Mention the builtin "help"

### ISSUE THREAD

**Issue by jalanb** [jlevy/the-art-of-command-line#32]
As the guide is so Bash-centric I suggest mentioning the builtin "help" command, as it can be the only source of extra info for some commands (other builtins)

For example, `disown` is mentioned so I tried to find out more about it

```shell
$ man disown
No manual entry for disown
$ disown --help
bash: disown: --: invalid option
disown: usage: disown [-h] [-ar] [jobspec ...]
$ disown -h
bash: disown: current: no such job
$ help disown
disown: disown [-h] [-ar] [jobspec ...]
    Remove jobs from current shell.
    
    Removes each JOBSPEC argument from the table of active jobs.  Without
    any JOBSPECs, the shell uses its notion of the current job.
    
    Options:
      -a	remove all jobs if JOBSPEC is not supplied
      -h	mark each JOBSPEC so that SIGHUP is not sent to the job if the
    	shell receives a SIGHUP
      -r	remove only running jobs
    
    Exit Status:
    Returns success unless an invalid option or JOBSPEC is given.
```

**Comment by jlevy** [comment]
Fixed in 23c653b99f2d0eb1fa2893e9007a33a4d8265005

**Comment by jlevy** [comment]
Thanks!
